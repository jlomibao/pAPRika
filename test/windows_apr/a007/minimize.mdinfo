

   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2747E+02     4.9066E-01     1.5809E+00     H32       105

 BOND    =        1.2628  ANGLE   =      107.5422  DIHED      =       52.6110
 VDWAALS =      -68.5641  EEL     =     1332.4208  EGB        =     -127.4485
 1-4 VDW =        5.5104  1-4 EEL =    -2130.8178  RESTRAINT  =        0.0169
 EAMBER  =     -827.4832
 NMR restraints: Bond =    0.002   Angle =     0.014   Torsion =     0.001
===============================================================================
