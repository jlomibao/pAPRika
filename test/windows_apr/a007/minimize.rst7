defa
  125
  -0.5295452   4.9542223   1.8217974   1.0068668   4.8908061   1.8224298
  -0.8903048   4.2756647   0.5843968  -0.8949926   4.2446819   3.0401076
  -0.9090029   5.9824832   1.8335627   1.4698093   5.8842730   1.8348659
   1.3117172   4.1850613   0.5853888   1.3119898   4.1530575   3.0405090
   2.6796653   4.0714441   0.1005900   2.6759048   4.0890050   3.5517349
   0.1928399   3.8250277   3.7609441   0.1965356   3.8780591  -0.1476113
   0.1726366   3.3287613   4.8740026   0.1781871   3.4083330  -1.2720838
   2.6633380   4.0211803  -0.9942382   3.2410379   4.9545720   0.4267561
   2.6399619   4.0589147   4.6468351   3.2171903   4.9825918   3.2201464
  -4.6606414   1.9818280   1.8299414  -3.8191209   3.2695280   1.8265253
  -4.2265904   1.2946701   0.6210909  -4.2757419   1.3362196   3.0779040
  -5.7379717   2.1823266   1.8054844  -4.4356969   4.1754500   1.8024861
  -3.0223317   3.1438219   0.6136435  -3.0670386   3.1800505   3.0720589
  -2.2624232   4.2744970   0.0978592  -2.2594849   4.2940305   3.5513235
  -3.3906315   2.0762603   3.8154989  -3.3017550   2.0060828  -0.0970619
  -3.0165172   1.8348549   4.9502534  -2.8686399   1.7202433  -1.1999122
  -2.2489574   4.2204851  -0.9968294  -2.7491618   5.2018215   0.4212678
  -2.2262996   4.2600194   4.6463960  -2.7245029   5.2298321   3.2204582
  -4.0939902  -3.0806200   1.8542335  -4.8213520  -1.7251586   1.8457030
  -3.3317847  -3.0579157   0.6120966  -3.2934417  -3.0216020   3.0699031
  -4.7860845  -3.9302824   1.8771713  -5.9121198  -1.8318486   1.8624801
  -4.3730921  -1.1151241   0.6008698  -4.3375007  -1.0775036   3.0574692
  -4.9488957   0.1327264   0.1194415  -4.9601737   0.1425824   3.5549263
  -3.4786769  -1.8639888   3.7793498  -3.5495207  -1.9283639  -0.1308857
  -3.0268401  -1.6136481   4.8833252  -3.1444094  -1.7171936  -1.2610248
  -4.9102117   0.1398148  -0.9759558  -5.9895742   0.1919472   0.4584390
  -4.9295659   0.1321246   4.6505519  -5.9997885   0.1732182   3.2089484
   0.5788813  -5.1034123   1.8656750  -0.9575333  -5.0388306   1.8659688
   0.9433744  -4.3990056   0.6441763   0.9415638  -4.4181028   3.0988366
   0.9580834  -6.1318031   1.8582904  -1.4216827  -6.0317837   1.8591799
  -1.2623943  -4.3066641   0.6444951  -1.2607473  -4.3251499   3.0991309
  -2.6262671  -4.2395888   0.1350252  -2.6280406  -4.2101905   3.5864574
  -0.1443337  -4.0129881   3.8282943  -0.1439005  -3.9774923  -0.0758928
  -0.1244092  -3.5333147   4.9485252  -0.1237180  -3.4772565  -1.1872142
  -2.5920153  -4.2119265  -0.9601933  -3.1702969  -5.1303126   0.4698103
  -2.6092417  -4.1561952   4.6810682  -3.1896742  -5.0947810   3.2647343
   4.7088799  -2.1282509   1.8458293   3.8688729  -3.4168931   1.8537203
   4.3132264  -1.4811056   0.6020074   4.2824968  -1.4426735   3.0583141
   5.7866458  -2.3272823   1.8617661   4.4861633  -4.3223992   1.8751280
   3.1100983  -3.3280516   0.6124610   3.0772343  -3.2911837   3.0701316
   2.3078486  -4.4457922   0.1341688   2.3136944  -4.4194941   3.5861165
   3.3604913  -2.1539682   3.7802463   3.4221203  -2.2200499  -0.1293971
   2.9316884  -1.8679743   4.8847507   3.0343808  -1.9743372  -1.2586437
   2.2756593  -4.4139057  -0.9609974   2.7754112  -5.3794411   0.4675206
   2.2996014  -4.3647791   4.6807622   2.7984831  -5.3482233   3.2638118
   4.1459613   2.9353005   1.8285616   4.8702864   1.5780300   1.8317057
   3.3430412   2.8807541   0.6144282   3.3885893   2.9113858   3.0739518
   4.8402867   3.7831368   1.8056610   5.9611507   1.6823452   1.8077954
   4.3775897   0.9321939   0.6223812   4.4290889   0.9678313   3.0789403
   4.9951455  -0.2882740   0.1199785   5.0072500  -0.2806260   3.5565320
   3.6111435   1.7819982   3.8160215   3.5184540   1.7220158  -0.0953744
   3.2149372   1.5719345   4.9495049   3.0602652   1.4735320  -1.1971450
   4.9560923  -0.2777850  -0.9753783   6.0373935  -0.3200843   0.4578328
   4.9746340  -0.2884310   4.6521218   6.0460653  -0.3397763   3.2118548
   0.0144045  -0.0852072  -0.0716668  -0.9011523  -0.6687633  -0.2278777
   0.8746787  -0.7465592  -0.2320606   0.0474751   0.7090368  -0.8274852
   0.0437091   0.5090640   1.3477285   0.9530679   1.1096819   1.4722903
  -0.0027579  -0.5929019   2.4258210  -0.8086207   1.1874768   1.4761305
  -0.9122041  -1.1933349   2.3009639   0.8494176  -1.2714729   2.2972134
   0.0266574   0.0003363   3.8456209  -0.8328860   0.6626110   4.0061197
  -0.0077551  -0.7946263   4.6006244   0.9429198   0.5825526   4.0026890
   0.0002656   0.0000025  -6.0009671  -0.0002536  -0.0013108  -9.0000677
  -0.0000952   2.2000034 -11.2000229
