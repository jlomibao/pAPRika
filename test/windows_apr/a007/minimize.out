
          -------------------------------------------------------
          Amber 18 SANDER                              2018
          -------------------------------------------------------

| Run on 04/10/2019 at 18:01:20

|   Executable path: sander
| Working directory: /home/jlomibao/pAPRika_fork/pAPRika/test/windows_apr/a007
|          Hostname: entropy.ucsd.edu

  [-O]verwriting output

File Assignments:
|  MDIN: minimize.in                                                           
| MDOUT: minimize.out                                                          
|INPCRD: cb6-but-dum.rst7                                                      
|  PARM: cb6-but-dum.prmtop                                                    
|RESTRT: minimize.rst7                                                         
|  REFC: cb6-but-dum.rst7                                                      
| MDVEL: mdvel                                                                 
| MDFRC: mdfrc                                                                 
|  MDEN: minimize.mden                                                         
| MDCRD: minimize.nc                                                           
|MDINFO: minimize.mdinfo                                                       
|  MTMD: mtmd                                                                  
|INPDIP: inpdip                                                                
|RSTDIP: rstdip                                                                
|INPTRA: inptraj                                                               


 Here is the input file:

PBC MD Simulation                                                              
 &cntrl                                                                        
  imin =          1,                                                           
  ntx =           1,                                                           
  irest =         0,                                                           
  maxcyc =        5000,                                                        
  ncyc =          5000,                                                        
  dt =            0.002,                                                       
  nstlim =        5000,                                                        
  ntpr =          500,                                                         
  ntwe =          500,                                                         
  ntwr =          5000,                                                        
  ntwx =          500,                                                         
  ntxo =          1,                                                           
  ioutfm =        1,                                                           
  ntf =           2,                                                           
  ntc =           2,                                                           
  cut =           999.0,                                                       
  igb =           1,                                                           
  tempi =         298.15,                                                      
  temp0 =         298.15,                                                      
  ntt =           3,                                                           
  gamma_ln =      1.0,                                                         
  ig =            -1,                                                          
  ntp =           0,                                                           
  barostat =      2,                                                           
  ntr =           1,                                                           
  restraint_wt =  50.0,                                                        
  restraintmask = '@DUM',                                                      
  nmropt =        1,                                                           
  pencut =        -1,                                                          
  ntb =           0,                                                           
 /                                                                             
 &wt type = 'END', /                                                           
DISANG = rest.in                                                               
LISTOUT = POUT                                                                 
                                                                               

--------------------------------------------------------------------------------
   1.  RESOURCE   USE: 
--------------------------------------------------------------------------------

| Flags:                                                                        
| New format PARM file being parsed.
| Version =    1.000 Date = 01/16/19 Time = 17:42:13
 NATOM  =     125 NTYPES =       7 NBONH =      46 MBONA  =      93
 NTHETH =     118 MTHETA =     158 NPHIH =     200 MPHIA  =     375
 NHPARM =       0 NPARM  =       0 NNB   =     836 NRES   =       5
 NBONA  =      93 NTHETA =     158 NPHIA =     375 NUMBND =       6
 NUMANG =      12 NPTRA  =      13 NATYP =       7 NPHB   =       0
 IFBOX  =       0 NMXRS  =     108 IFCAP =       0 NEXTRA =       0
 NCOPY  =       0

 Implicit solvent radii are modified Bondi radii (mbondi)                                                   

|     Memory Use     Allocated
|     Real               14705
|     Hollerith            382
|     Integer            27997
|     Max Pairs              1
|     nblistReal             0
|     nblist Int             0
|       Total              225 kbytes

| Note: 1-4 EEL scale factors are being read from the topology file.

| Note: 1-4 VDW scale factors are being read from the topology file.
| Duplicated    0 dihedrals
| Duplicated    0 dihedrals
Note: ig = -1. Setting random seed to   173211 based on wallclock time in microseconds.

--------------------------------------------------------------------------------
   2.  CONTROL  DATA  FOR  THE  RUN
--------------------------------------------------------------------------------

default_name                                                                    

General flags:
     imin    =       1, nmropt  =       1

Nature and format of input:
     ntx     =       1, irest   =       0, ntrx    =       1

Nature and format of output:
     ntxo    =       1, ntpr    =     500, ntrx    =       1, ntwr    =    5000
     iwrap   =       0, ntwx    =     500, ntwv    =       0, ntwe    =     500
     ioutfm  =       1, ntwprt  =       0, idecomp =       0, rbornstat=      0

Potential function:
     ntf     =       2, ntb     =       0, igb     =       1, nsnb    =      25
     ipol    =       0, gbsa    =       0, iesp    =       0
     dielc   =   1.00000, cut     = 999.00000, intdiel =   1.00000
     saltcon =   0.00000, offset  =   0.09000, gbalpha=    1.00000
     gbbeta  =   0.00000, gbgamma =   0.00000, surften =   0.00500
     rdt     =   0.00000, rgbmax  =  25.00000  extdiel =  78.50000
     alpb  =        0

Frozen or restrained atoms:
     ibelly  =       0, ntr     =       1
     restraint_wt =  50.00000

Energy minimization:
     maxcyc  =    5000, ncyc    =    5000, ntmin   =       1
     dx0     =   0.01000, drms    =   0.00010

SHAKE:
     ntc     =       2, jfastw  =       0
     tol     =   0.00001

NMR refinement options:
     iscale  =       0, noeskp  =       1, ipnlty  =       1, mxsub   =       1
     scalm   = 100.00000, pencut  =  -1.00000, tausw   =   0.10000
|  INFO: Old style inpcrd file read


    LOADING THE CONSTRAINED ATOMS AS GROUPS


   5.  REFERENCE ATOM COORDINATES

  defa
     Mask @DUM; matches     3 atoms

--------------------------------------------------------------------------------
   3.  ATOMIC COORDINATES AND VELOCITIES
--------------------------------------------------------------------------------

default_name                                                                    
 begin time read from input coords =     0.000 ps



           Begin reading energy term weight changes/NMR restraints
 WEIGHT CHANGES:
                         ** No weight changes given **

 RESTRAINTS:
 Requested file redirections:
  DISANG    = rest.in
  LISTOUT   = POUT
 Restraints will be read from file: rest.in
Here are comments from the DISANG input file:

                       Number of restraints read =     9

                  Done reading weight changes/NMR restraints


 Number of triangulated 3-point waters found:        0

--------------------------------------------------------------------------------
   4.  RESULTS
--------------------------------------------------------------------------------



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
      1      -8.1356E+02     1.0057E+01     2.4770E+01     N12        57

 BOND    =       11.0328  ANGLE   =      109.3438  DIHED      =       52.9822
 VDWAALS =      -67.7110  EEL     =     1326.2731  EGB        =     -123.8292
 1-4 VDW =        5.8710  1-4 EEL =    -2127.5276  RESTRAINT  =        0.0000
 EAMBER  =     -813.5649
 NMR restraints: Bond =    0.000   Angle =     0.000   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
    500      -8.2743E+02     4.9801E-01     1.6771E+00     H7        120

 BOND    =        1.2505  ANGLE   =      107.5594  DIHED      =       52.5864
 VDWAALS =      -68.4347  EEL     =     1332.0512  EGB        =     -127.3782
 1-4 VDW =        5.4817  1-4 EEL =    -2130.5476  RESTRAINT  =        0.0038
 EAMBER  =     -827.4313
 NMR restraints: Bond =    0.001   Angle =     0.003   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1000      -8.2745E+02     4.9383E-01     1.5809E+00     H32       105

 BOND    =        1.2637  ANGLE   =      107.5461  DIHED      =       52.6112
 VDWAALS =      -68.5207  EEL     =     1332.3213  EGB        =     -127.4351
 1-4 VDW =        5.5025  1-4 EEL =    -2130.7431  RESTRAINT  =        0.0082
 EAMBER  =     -827.4541
 NMR restraints: Bond =    0.001   Angle =     0.006   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   1500      -8.2746E+02     4.9661E-01     1.5809E+00     H32       105

 BOND    =        1.2739  ANGLE   =      107.5429  DIHED      =       52.6115
 VDWAALS =      -68.5435  EEL     =     1332.3970  EGB        =     -127.4642
 1-4 VDW =        5.5053  1-4 EEL =    -2130.7905  RESTRAINT  =        0.0116
 EAMBER  =     -827.4676
 NMR restraints: Bond =    0.002   Angle =     0.009   Torsion =     0.000
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2000      -8.2746E+02     4.9607E-01     1.5808E+00     H32       105

 BOND    =        1.2504  ANGLE   =      107.5446  DIHED      =       52.6104
 VDWAALS =      -68.5551  EEL     =     1332.3820  EGB        =     -127.4214
 1-4 VDW =        5.5114  1-4 EEL =    -2130.7990  RESTRAINT  =        0.0144
 EAMBER  =     -827.4767
 NMR restraints: Bond =    0.002   Angle =     0.012   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   2500      -8.2747E+02     4.9071E-01     1.5809E+00     H32       105

 BOND    =        1.2628  ANGLE   =      107.5422  DIHED      =       52.6109
 VDWAALS =      -68.5634  EEL     =     1332.4196  EGB        =     -127.4485
 1-4 VDW =        5.5103  1-4 EEL =    -2130.8166  RESTRAINT  =        0.0167
 EAMBER  =     -827.4826
 NMR restraints: Bond =    0.002   Angle =     0.013   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3000      -8.2747E+02     4.9066E-01     1.5809E+00     H32       105

 BOND    =        1.2628  ANGLE   =      107.5422  DIHED      =       52.6110
 VDWAALS =      -68.5640  EEL     =     1332.4208  EGB        =     -127.4485
 1-4 VDW =        5.5104  1-4 EEL =    -2130.8177  RESTRAINT  =        0.0169
 EAMBER  =     -827.4832
 NMR restraints: Bond =    0.002   Angle =     0.014   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   3500      -8.2747E+02     4.9066E-01     1.5809E+00     H32       105

 BOND    =        1.2628  ANGLE   =      107.5422  DIHED      =       52.6110
 VDWAALS =      -68.5640  EEL     =     1332.4208  EGB        =     -127.4485
 1-4 VDW =        5.5104  1-4 EEL =    -2130.8178  RESTRAINT  =        0.0169
 EAMBER  =     -827.4832
 NMR restraints: Bond =    0.002   Angle =     0.014   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4000      -8.2747E+02     4.9066E-01     1.5809E+00     H32       105

 BOND    =        1.2628  ANGLE   =      107.5422  DIHED      =       52.6110
 VDWAALS =      -68.5640  EEL     =     1332.4209  EGB        =     -127.4485
 1-4 VDW =        5.5103  1-4 EEL =    -2130.8179  RESTRAINT  =        0.0169
 EAMBER  =     -827.4833
 NMR restraints: Bond =    0.002   Angle =     0.014   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   4500      -8.2747E+02     4.9066E-01     1.5809E+00     H32       105

 BOND    =        1.2628  ANGLE   =      107.5422  DIHED      =       52.6110
 VDWAALS =      -68.5640  EEL     =     1332.4209  EGB        =     -127.4485
 1-4 VDW =        5.5103  1-4 EEL =    -2130.8179  RESTRAINT  =        0.0169
 EAMBER  =     -827.4833
 NMR restraints: Bond =    0.002   Angle =     0.014   Torsion =     0.001
===============================================================================


   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2747E+02     4.9066E-01     1.5809E+00     H32       105

 BOND    =        1.2628  ANGLE   =      107.5422  DIHED      =       52.6110
 VDWAALS =      -68.5641  EEL     =     1332.4208  EGB        =     -127.4485
 1-4 VDW =        5.5104  1-4 EEL =    -2130.8178  RESTRAINT  =        0.0169
 EAMBER  =     -827.4832
 NMR restraints: Bond =    0.002   Angle =     0.014   Torsion =     0.001
===============================================================================


  Maximum number of minimization cycles reached.


                    FINAL RESULTS



   NSTEP       ENERGY          RMS            GMAX         NAME    NUMBER
   5000      -8.2747E+02     4.9066E-01     1.5809E+00     H32       105

 BOND    =        1.2628  ANGLE   =      107.5422  DIHED      =       52.6110
 VDWAALS =      -68.5641  EEL     =     1332.4208  EGB        =     -127.4485
 1-4 VDW =        5.5104  1-4 EEL =    -2130.8178  RESTRAINT  =        0.0169
 EAMBER  =     -827.4832
 NMR restraints: Bond =    0.002   Angle =     0.014   Torsion =     0.001
===============================================================================
 ------------------------------------------------------------------------------


 Final Restraint Analysis for coords: minimize.rst7                           


 Restraints, deviations, and energy contributions:    pencut =   -1.00

 ------------------------------------------------------------------------------
     First atom        Last atom    curr. value target deviation  penalty
 ------------------------------------------------------------------------------
  DUM  DM1    3 --  C7   CB6    1:    9.303    9.302    0.001    0.000 d    0: 0
  DUM  DM2    4 --  C7   CB6    1:  147.296  147.398    0.102    0.000 a
  DUM  DM2    4 --  DUM  DM1    3:   49.489   49.483    0.006    0.000 t
  DUM  DM1    3 --  C31  CB6    1:   61.989   62.003    0.014    0.000 a
  DUM  DM1    3 --  C7   CB6    1: -146.088 -146.178    0.091    0.000 t
  C7   CB6    1 --  C31  CB6    1:  -72.707  -72.548    0.158    0.001 t
  DUM  DM1    3 --  C    BUT    2:    5.930    6.000    0.070    0.002 d    0: 0
  DUM  DM2    4 --  C    BUT    2:  179.142  180.000    0.858    0.002 a
  DUM  DM1    3 --  C3   BUT    2:  177.925  180.000    2.075    0.011 a
                                       Total distance penalty:      0.002
                                       Total angle    penalty:      0.014
                                       Total torsion  penalty:      0.001
|                               RMS deviation from ideal bonds :      0.0096
|                               RMS deviation from ideal angles:      4.465
 ------------------------------------------------------------------------------

--------------------------------------------------------------------------------
   5.  TIMINGS
--------------------------------------------------------------------------------

|                Calc gb radii              2.38 (22.39% of Gen B)
|                Calc gb diag               4.23 (39.74% of Gen B)
|                Calc gb off-diag           4.03 (37.85% of Gen B)
|                Other                      0.00 ( 0.02% of Gen B)
|             Gen Born time             10.64 (100.0% of Nonbo)
|          Nonbond force             10.64 (90.63% of Force)
|          Bond/Angle/Dihedral        1.06 ( 9.07% of Force)
|          Other                      0.04 ( 0.30% of Force)
|       Force time                11.74 (100.0% of Runmd)
|    Runmd Time                11.74 (99.77% of Total)
|    Other                      0.03 ( 0.22% of Total)
| Total time                11.77 (100.0% of ALL  )

| Number of list builds   :          0

| Highest rstack allocated:          0
| Highest istack allocated:          0
|           Job began  at 18:01:20.169  on 04/10/2019
|           Setup done at 18:01:20.174  on 04/10/2019
|           Run   done at 18:01:31.937  on 04/10/2019
|     wallclock() was called   90028 times

|3D-RISM memory allocation summary
|Type          Maximum        Current   
|Integer       0.00000 GB     0.00000 GB
|Real          0.00000 GB     0.00000 GB
|Logical       0.00000 GB     0.00000 GB
|Character     0.00000 GB     0.00000 GB
|---------------------------------------
|Total         0.00000 GB     0.00000 GB
